x/bank/client/rest/tx.go

```diff
@@ -18,7 +20,52 @@ type SendReq struct {
 	Amount  sdk.Coins    `json:"amount" yaml:"amount"`
 }
 
+// NewSendRequestHandlerFn returns an HTTP REST handler for creating a MsgSend
+// transaction.
+func NewSendRequestHandlerFn(ctx context.CLIContext, m codec.Marshaler, txg tx.Generator) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		ctx = ctx.WithMarshaler(m)
+
+		vars := mux.Vars(r)
+		bech32Addr := vars["address"]
+
+		toAddr, err := sdk.AccAddressFromBech32(bech32Addr)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+			return
+		}
+
+		var req SendReq
+		if !rest.ReadRESTReq(w, r, ctx.Marshaler, &req) {
+			return
+		}
+
+		req.BaseReq = req.BaseReq.Sanitize()
+		if !req.BaseReq.ValidateBasic(w) {
+			return
+		}
+
+		fromAddr, err := sdk.AccAddressFromBech32(req.BaseReq.From)
+		if err != nil {
+			rest.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
+			return
+		}
+
+		msg := types.NewMsgSend(fromAddr, toAddr, req.Amount)
+		tx.WriteGeneratedTxResponse(ctx, w, txg, req.BaseReq, msg)
+	}
+}
+
+// ---------------------------------------------------------------------------
+// Deprecated
+//
+// TODO: Remove once client-side Protobuf migration has been completed.
+// ---------------------------------------------------------------------------
```

Comment:
```suggestion
```

Reply:
??? I'm not going to change all of these. I've been using the same template since starting the migration.